serverless.yml

```diff
@@ -12,6 +12,9 @@ provider:
   environment:
     COLLECTION_TEMPLATES_BUCKET_NAME: ${self:custom.collectionTemplatesBucketName}
     EDL_PASSWORD: ${env:EDL_PASSWORD, ''}
+    COOKIE_DOMAIN: ${env:COOKIE_DOMAIN, 'localhost'}
+    JWT_SECRET: ${env:JWT_SECRET, 'local-secret'}
+    JWT_VALID_TIME: ${env:JWT_VALID_TIME, '900'}
```

Comment:
Is this set differently in CI or are we purposely expiring the JWT faster than the LP token?